test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Get-Counter.Tests.ps1

```diff
@@ -241,3 +241,19 @@ Describe "Feature tests for Get-Counter cmdlet" -Tags "Feature" {
         }
     }
 }
+
+Describe "Get-Counter cmdlet does not run on IoT" -Tags "CI" {
+
+    It "Get-Counter throws PlatformNotSupportedException" -Skip:$(-not [System.Management.Automation.Platform]::IsIoT)  {
```

Comment:
We skip all counter test for Linux, IOS and IoT. Why do we test the throw only for IoT?